### PR TITLE
AP-26 created mock cart, image, product name, cost, quantity counter,…

### PR DESCRIPTION
**Pull request by istuart50**
… delete button, styling
https://apollodev23.atlassian.net/browse/AP-26

### CODE REVIEW

File: products.html [hunk 1]
```diff
@@ -17,8 +17,11 @@
 
 <body>
 
+
```
Comment (GrantGriffin) [inline products.html:20]: lets watch out for useless changes to other files.  This can increase risk for merge conflicts

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.cart-container {
+    /* min-width: 250px; */
+    /* max-width: 500px; */
+    /* width: 300px; */
```
Comment (GrantGriffin) [inline css/styles.css]: Remove commented code

File: products.html (hunk 1 above)
Comment (GrantGriffin) [reply products.html:20]: It's okay to leave this one for now.

**Review by GrantGriffin — CHANGES_REQUESTED** [review]
Please make a commit to fix the comments before merging this.

File: css/styles.css (hunk 2 above)
Comment (istuart50) [reply css/styles.css]: Done

**Review by GrantGriffin — APPROVED** [review]

**Pull request merged**